libs/status/crate/src/lib.rs

```diff
@@ -0,0 +1,52 @@
+//! Defines a logical status and error model that is suitable for different programming
+//! environments, including REST APIs and RPC APIs.
+
+mod gen;
+mod status_code;
+
+use std::fmt::Debug;
+
+use serde::{Deserialize, Serialize};
+pub use status_code::StatusCode;
+
+/**
+ * The canonical shape of a response object describing the status of a request between services.
+ */
```

Comment:
Is this a limitation of the generator that we don't get proper docstrings for `jsdoc` comments on types?

Reply:
Nope, this was just me being silly and copy pasting and not changing the comment format 😅